.github/workflows/continuous-integration.yml

```diff
@@ -6,28 +6,66 @@ on:
 
 jobs:
   static-analysis-psalm:
-    name: Static Analysis with Psalm
-    runs-on: ubuntu-latest
+    name: "Static Analysis with Psalm"
```

Comment:
Are changes like this really needed? I'd rather keep it simple and only add quotes where required (e.g. strings with special characters, numeric strings that must remain strings, etc.).

This file will be maintained by humans who will not see the need to add quotes everywhere.

Reply:
It is trying to maintain consistency across repos
https://github.com/doctrine/orm/pull/8116/files

Reply:
So what's the reason why we want to have all strings quoted in all repositories?

Reply:
i suspect inspiration from https://github.com/ergebnis/php-library-template/blob/master/.github/workflows/integrate.yaml

Reply:
it's just coding style, and that repo enforces it:
https://github.com/ergebnis/php-library-template/blob/master/.yamllint.yaml

I personally don't like everything quoted, as you point out it's not necessary all the time. but being consistent is the main thing i guess.

cc @localheinz

Reply:
@morozov 

I am using [`ibiqlik/action-yamllint`](https://github.com/ibiqlik/action-yamllint) to lint YAML files using [`yamllint`](https://github.com/adrienverge/yamllint), and for consistency, I quote YAML strings.

As @bendavies pointed out, it's a matter of taste and consistency. I have to admit that I dislike having to escape a bunch of characters when quoting.